core/ecschema-locaters/src/SchemaFileUtility.ts

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as fs from "fs-extra";
+import * as path from "path";
+import { DOMParser, XMLSerializer } from "@xmldom/xmldom";
+import { Schema } from "@itwin/ecschema-metadata";
+
+/**
+ * Utility class to assist in creating serialized EC Schemas on the file system.
+ */
```

Comment:
missing release tag (presumably alpha).

Reply:
"alpha" means "nobody else should be using this API yet". If you want people to try it out, it should be "beta". [Release tag guidelines](https://www.itwinjs.org/learning/guidelines/release-tags-guidelines/).
Personally I see little use for "alpha" APIs.

Reply:
Then we should make this API beta.

Reply:
In that case it's time to start defining documentation groups for this package and add it to the [API reference](https://www.itwinjs.org/reference/). Separate PR perhaps.

Reply:
@pmconne would showing it in the documentation make it more useful? I see it as something very experimental that we will probably change, whereas `beta` would be something we think we have the right API/contract but want user validation/feedback first.

Either way, I think both should be in the docs with just that in mind.

Reply:
@christophermlawson is working on documentation, we do plan to put it in another PR.

Reply:
@calebmshafer do you understand how extract-api is succeeding without release tag on this exported class?

Reply:
All the classes in the package are alpha.  Should I change them all to beta?

Reply:
I'm assuming yes.  All classes are beta now.

Reply:
and api extract done